doc/modules/mod_inbox.md

```diff
@@ -28,42 +28,92 @@ Inbox currently supports the following DBs:
 * PgSQL via native driver
 * MSSQL via ODBC driver
 
+### Filtering and ordering
+
+Inbox query results may be filtered by time range and sorted by timestamp.
+By default, `mod_inbox` returns all conversations, most recently updated first.
```

Comment:
By default, `mod_inbox` returns all conversations, listing the ones updated most recently first?